style.css

```diff
@@ -110,3 +112,487 @@ ul {
+  background-repeat: no-repeat;
+  line-height: 75px;
+  background-size: 327px 386px;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.card1 {
+  order: 6;
+}
+
+.card2 {
+  order: 7;
+}
+
+.card3 {
+  order: 7;
+}
+
+.card4 {
+  order: 7;
+}
+
+.card5 {
+  order: 7;
+}
+
+.card6 {
+  order: 7;
+}
+
+.cardt {
+  display: flex;
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-top: 30%;
+  padding-left: 5%;
+}
+
+.cardp {
+  display: flex;
+  padding-left: 5%;
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.cardb {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  column-gap: 10px;
+  margin: 0 0 30px 10px;
+}
+
+.bot {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.button1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 40px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  background-size: 10px 20px;
+}
+
+.button2 {
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 95px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+a {
+  text-decoration: none;
+}
+
+.button3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.link-button2 {
+  display: flex;
+  align-items: center;
+  margin-bottom: 10%;
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+}
+
+.About-myself {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  background: #1c1a19;
+}
+
+.about-text {
+  width: 327px;
+  height: 79px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  align-self: center;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  padding-top: 10%;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.regular-text2 {
+  align-self: center;
+  width: 327px;
+  height: 148px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.resume {
+  display: flex;
+  padding: 12px;
+  width: 159px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  margin-left: 6%;
+}
+
+.line2 {
+  margin: 50% 0 20%;
+  border: 1px solid #fff;
+  align-self: center;
+  width: 80%;
+}
+
+.lang {
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly align the languages to the center and space them as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205450731-6e25972c-4be3-490b-9acb-5201f8c22582.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205450709-9eb813f6-8127-4801-b3dd-312452afe25b.png)

Reply:
done